internal/page/donor/your_address.go

```diff
@@ -42,6 +55,11 @@ func YourAddress(logger Logger, tmpl template.Template, addressClient AddressCli
 						return err
 					}
 
+					if makingAnotherLPA {
+						r.Form.Del("from")
```

Comment:
I'd just use a different param for making another, that way `from` can keep working the same way